challenge_4/python/OkraArmband/src/challenge_4.py

```diff
@@ -0,0 +1,49 @@
+class BinaryTree():
+    def __init__(self, root):
+        self.root = root
+        self.left = None
+        self.right = None
+
+    def invertTree(self):
+        """
+        Swap left and right children of input BinaryTree for all subsequent children as well (recursively).
+        """
+        temp = self.left
+        self.left = self.right
+        self.right = temp
+
+
+        # Continue the recursion as long as we haven't reached the end of a branch
+        if self.left is not None:
```

Comment:
This is more of a syntaxtic thing but you could just say `if self.left`. That's a check for whether or not `self.left` is None or not.